.travis.yml

```diff
@@ -8,6 +8,7 @@ before_install:
   - nix-env -iA cachix -f https://cachix.org/api/v1/install
   - cachix use cachix
   - cachix use dapp
-  - nix-env -f https://github.com/dapphub/dapptools/tarball/master -iA dapp
+  - git clone --recursive https://github.com/dapphub/dapptools $HOME/.dapp/dapptools
+  - nix-env -f $HOME/.dapp/dapptools -iA dapp solc
 script:
-  - dapp test
+  - dapp --use solc:0.5.11 test
```

Comment:
In order to use a specific version of `solidity` with `dapp` we need `$HOME/.dapp/dapptools` populated.  This solves the problem, and is how we do it in the `dss` repo; however, it did appear to add a noticeable increase to CI runtime.  If we were constantly changing this code, it could be painful.  I'm calling it out incase someone thinks of an easy fix, and because if there is an easy fix we can probably apply it to `dss` too.

Reply:
Looks like the time delay is in:
`nix-env -f $HOME/.dapp/dapptools -iA dapp solc`